tests/python/gaia-ui-tests/gaiatest/apps/clock/app.py

```diff
@@ -27,9 +27,18 @@ def launch(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._alarm_create_new_locator))))
 
-    @property
-    def alarms(self):
-        return [self.Alarm(self.marionette, alarm) for alarm in self.marionette.find_elements(*self._all_alarms_locator)]
+    def switch_view(self, view_name):
+        Wait(self.marionette).until(
+            expected.element_present(*(By.ID, self._clock_views.get(view_name)))).tap()
+        if view_name == 'stopwatch':
+            from gaiatest.apps.clock.regions.stopwatch import StopWatch
+            return StopWatch(self.marionette)
+        elif view_name == 'timer':
+            from gaiatest.apps.clock.regions.timer import Timer
+            return Timer(self.marionette)
+        else:
```

Comment:
I thought of using an Enum but it seems like overkill. Especially since this is the only place it would be used. If we were going to be using it throughout our whole suite, I think an enum would be more appropriate. What say you @JohanLorenzo?